.github/workflows/publish.yaml

```diff
@@ -20,14 +20,68 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node_version:
-          - '12'
-          - '14'
-          - '16'
-        os:
-          - macos-10.15
-          - ubuntu-20.04
-          - windows-2019
+        include:
```

Comment:
Would this be easier to read if you kept the original matrix, then removed everything from `include` except the three arm64 targets?

Reply:
Yes, if there was a way to make `target` still work with that. I couldn't find a way to, which is why I ended up here.

Reply:
:( Okay, thanks for checking.

Reply:
I tried something different, seems like it's working and it's a bit easier to follow, lmk if that works for you.

Reply:
I also didn't :D